modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -0,0 +1,128 @@
+= Migrating User Key-based Encryption to Master Key Encryption
```

Comment:
This sentence should end as "to Master Key-based Encryption"

Reply:
done